src/components/Swap.tsx

```diff
@@ -323,6 +324,10 @@ const Swap = ({ transferChains }: SwapProps) => {
 
   useEffect(() => {
     const load = async () => {
+      LiFi.setConfig({
```

Comment:
Please also set the api uri based on the environment the react app is deployed in.